node-liveness-checker/Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+pipeline {
+    agent any
+    environment {
+        aws_s3_bucket_name = 's3://node-deb.concordium.com'
+    }
+    stages {
+        stage('build') {
+            agent {
+                dockerfile {
```

Comment:
To build with the `dockerfile` agent you need a `Dockerfile`. It would make sense to just steal [this one](https://gitlab.com/Concordium/infra/custom-node-exporter/-/blob/master/Dockerfile) (or, in case `rustfmt` is no longer needed, just `${rust_image_tag}` on a `docker` agent).